src/include/artifact-download-data.h

```diff
@@ -17,16 +17,16 @@
  * limitations under the License.
  */
 
-#ifndef __MENDER_ARTIFACT_DOWNLOAD_DATA_H__
-#define __MENDER_ARTIFACT_DOWNLOAD_DATA_H__
+#ifndef __MENDER_ARTIFACT_DOWNLOAD_DATA_PRIV_H__
+#define __MENDER_ARTIFACT_DOWNLOAD_DATA_PRIV_H__
```

Comment:
It is a good call to add the `_PRIV_` to avoid collisions.

One small thing slightly confusing is that for the headers that have both public and private versions (example, `alloc.h`) we have the explicit `(private API)` in the doc line and the  `_PRIV_` in the guard, while for the headers that are only private, like the one I am commenting on, we only have the guard modified.

If it's just one more ninja command to run, could we add the `(private API)` in the doc line also in these?

Reply:
Good idea. I actually didn't even have the `_PRIV_` in the guard in those private-only headers. But that complicated the `check_include_headers.h` check too much. So I decided to add it to all private headers because it cannot hurt. :) Adding the `(private API)` part is easy and it can potentially be helpful to people looking at the sources.